js/curation-tool.js

```diff
@@ -818,8 +860,22 @@ const vm = new Vue({
               }
             }
           }
+        })
+        .style_edges((element, data) => {
+          // Is the parent a descendant of a pinning node? If so, we need to
+          // select this branch!
+          // console.log('Found an edge with data: ', data);
+          if (
+            hasProperty(data, 'source') &&
+            hasProperty(data.source, '@id') &&
+            pinningNodeChildrenIRIs.has(data.source['@id'])
+          ) {
+            // Apply a class to this branch.
+            element.classed('descendant-of-pinning-node-branch', true);
+          }
         });
-      tree(d3.layout.newick_parser(newick));
+      const countPhylogeny = this.testcase.phylogenies.indexOf(phylogeny) + 1;
+      tree(new PhylogenyWrapper(phylogeny).getParsedNewickWithIRIs(`http://example.org/produced_by_curation_tool#phylogeny${countPhylogeny}`));
```

Comment:
So there's two issues here:
- We shouldn't be making up the base URI at multiple locations. I've centralized top-level base URIs to PHYXWrapper in 81cdd0090 so we avoid doing that.
- However, I think you'd also like to see us using blank nodes instead of using a base URI. I've filed that as #92. Do you think it's okay if we use example.org in this pull request and fix it later, or should we close that issue before merging this PR?